README.md

```diff
@@ -1,4 +1,99 @@
+[![Build Status](https://travis-ci.org/kaeuferportal/facebook_ads.svg?branch=master)](https://travis-ci.org/kaeuferportal/facebook_ads)
+[![Code Climate](https://codeclimate.com/github/kaeuferportal/facebook_ads/badges/gpa.svg)](https://codeclimate.com/github/kaeuferportal/facebook_ads)
+[![Test Coverage](https://codeclimate.com/github/kaeuferportal/facebook_ads/badges/coverage.svg)](https://codeclimate.com/github/kaeuferportal/facebook_ads/coverage)
+
 # Facebook Ads
 
-This is a stub for a ruby gem to communicate with [Facebook's Marketing API](https://developers.facebook.com/docs/marketing-apis),
+This is a ruby gem to communicate with [Facebook's Marketing API](https://developers.facebook.com/docs/marketing-apis),
 especially the [Targeting of Ads](https://developers.facebook.com/docs/marketing-api/buying-api/targeting).
+
+## Installation
+
+TBD, once we are on RubyGems.org
+
+## Configuration
+
+First configure your access token:
+````ruby
+FacebookAds.configure do |config|
+  config.access_token = 'your_very_secret_token'
+end
+````
+
+If you prefer to load the config from a hash, you can do that too:
+````ruby
+FacebookAds.configure do |config|
+  config_hash = YAML.load_file('my_config.yml')
+  config.load_hash(config_hash)
+end
+````
+
+## Usage
+
+Then you can instantiate an empty model, which can later selectively
```

Comment:
Instantiate an empty model sounds weird. Maybe use '... instantiate a model (without data) which can later ...' or something like that?